### ISSUE TITLE
Security Page

### ISSUE THREAD

**Issue by janpio** [apache/cordova-docs#860]
Cordova is currently missing a Security page at https://cordova.apache.org/security/ or similar where the process of reporting security bugs is documented.

From what I understood, the current process consists of creating an issue via and setting "Security Level" to "Non-Public".

The page could list or link to previous CVEs:
https://cordova.apache.org/announcements/2015/11/20/security.html
https://www.cvedetails.com/vulnerability-list/vendor_id-45/product_id-27153/Apache-Cordova.html

Some examples:
https://nodejs.org/en/security/
https://www.ruby-lang.org/en/security/

**Comment by brodybits** [comment]
I would mark this one as a blocker.

**Comment by brodybits** [comment]
<https://httpd.apache.org/security/> - a classic!

**Comment by janpio** [comment]
> I would mark this one as a blocker.

I don't agree. There is a link to http://www.apache.org/security/ in the footer, see the updated description. The page itself on cordova.apache.org would just be a convenience "feature".
